src/main/resources/db/demo/kdt/19_kb_bid_review.sql

```
@@ -0,0 +1,46 @@
+- 영업담당자 정과 부의 개인정보동의서와 신분증 사본이 아직 모이지 않았다
+
+**주의할 점**
+
+기술능력 평가에서 최고점과 최저점을 뺀 산술평균이 배점의 80퍼센트에 못 미치거나 종합 점수가 85점에 못 미치면 협상적격에서 빠진다. 협상적격자 중 상위 두 곳만 우선협상대상이 된다.
+
+가격 배점은 고용보험 환급과정 비율이 40퍼센트를 넘어야 만점이다. 비환급 과정은 환급금에 상당하는 할인율을 제시해야 환급과정으로 인정된다.',
+ '2025-11-21 15:40:00', '2025-12-21 15:40:00');
+
+
+-- 확정 요약에 프로젝트 연결 (12 에서는 project 가 없어 NULL 이었다)
+UPDATE bid_notice_summary SET project_id = 8011 WHERE bid_notice_summary_id = 8011;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)